<launch>
    <node pkg="norlab_controllers_ros" exec="controller_node" name="controller_node" output="screen">
        <remap from="odom_in" to="icp_odom"/>
        <remap from="cmd_vel_out" to="cmd_vel"/>
        <param name="controller_config" value="$(find-pkg-share norlab_controllers_ros)/warthog-differential-orthexp.yaml"/>
    </node>
</launch>
